src/sagemaker/model.py

```diff
@@ -805,6 +823,8 @@ def delete_model(self):
             )
         self.sagemaker_session.delete_model(self.name)
 
+    destroy = delete_model
```

Comment:
Why not name the abstract method delete_model? Same for the predictor.

Reply:
Conciseness. We could use `Model.delete_model` and `Predictor.delete_predictor`, but IMO that's unnecessarily verbose. 

Ultimately, I think this issue is sort-of bike shedding. If you feel strongly about this suggestion (or if there are other factors I'm not considering), I'm happy to change it. Otherwise, we can just disagree and commit

Reply:
So its destroy vs delete_model, I wouldn't say its verbose, more explicit rather. But more than anything because we choose to have a different name, it just adds another lookup in the code "destroy = delete_model".

Reply:
I do think that while too long variable names are a code smell, given the context, in this scenario delete_model offers itself to better readability without trading the verbosity too much.

Reply:
Yeah, that makes sense to me. I hadn't really considered those as much. I'll make the change

Reply:
Addressed